README.md

```diff
@@ -98,7 +98,7 @@ For example, if the `jsonPath` expression was `$[*].id` and the command sent the
 ]
 ```
 
-Then then output would have the following contents:
+Then output would have the following contents:
```

Comment:
I think the intention was to say, `Then the output...`  (previously a typo in the second `then`)